deps/lz4/Makefile

```
@@ -0,0 +1,28 @@
+liblz4.a: lz4.o lz4hc.o lz4frame.o xxhash.o
+	$(AR) $(ARFLAGS) $@ $^
+
+lz4.o: lz4.c lz4.h
+lz4hc.o: lz4hc.c lz4hc.h lz4.h
+lz4frame.o: lz4frame.c lz4frame.h lz4.h lz4hc.h xxhash.h
+xxhash.o: xxhash.c xxhash.h
+
+.c.o:
+	$(R_CC) -fPIC -c $<
+
+clean:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)